packages/runner-shared/src/event-manager.js

```diff
@@ -289,10 +298,15 @@ const eventManager = {
   setup (config) {
     Cypress = this.Cypress = $Cypress.create(config)
 
+    // TODO: this was CT only before. Does this make sense for e2e too?
     // expose Cypress globally
-    window.Cypress = Cypress
+    // since CT AUT shares the window with the spec, we don't want to overwrite
+    // our spec Cypress instance with the component's Cypress instance
+    if (window.top === window) {
+      window.Cypress = Cypress
+    }
 
-    this._addListeners()
+    this._addListeners(Cypress)
```

Comment:
Not entirely sure what this does. `event-manager.js` had some ever so slight differences in E2E and CT, I took a ton  of time and care to merge them. This doesn't seem to break anything - anyone have some more context? Happy to dive deeper if anyone senses some risk.